tests/framework/installer/install_helper.go

```diff
@@ -341,6 +337,8 @@ func NewK8sRookhelper(clientset *kubernetes.Clientset, t func() *testing.T) *Ins
 	if err != nil {
 		panic("failed to get kubectl client :" + err.Error())
 	}
+	env := objects.E
+	flag.Parse()
```

Comment:
this `flag.Parse()` call will happen multiple times here, right?  that goes against what this comment says: https://github.com/rook/rook/issues/1316#issuecomment-351879286.  But, the golang docs don't say that it cannot be called more than once: https://golang.org/pkg/flag/#Parse, so maybe that's OK

Reply:
That sounds reasonable, thanks for the explanation.